docs/studio/webhooks.md

```diff
@@ -0,0 +1,234 @@
+# Webhooks in Studio
+
+## About webhooks
+
+Webhooks provide a way for the notifications to be delivered to an external web server whenever certain events occur in [Studio](https://studio.datachain.ai).
+
+Webhooks are used to receive data as it happens as opposed to polling an API to see if data is available. With webhooks, you can set a setting once that you want to hear about certain events or activities when you create the webhook.
+
+When you create a webhook, you specify a URL, and necessary information you want us to send to the URL along with the events that you want to listen on Datachain. When the event your webhook is subscribed to occurs, Datachain Studio will send an HTTP request with the data about the event to the URL that you specified. If your server is setup to listen for webhook deliveries at that URL, it can take action when it receives one.
+
+For example, you can subscribe your webhook to events that occur when a job is created, is complete, is failed, is running, and so on. You can then monitor whenever a job is failed through this webhook.
+
+### Alternative
+
+As opposed to webhooks, you can also use CLI command to get the job information
+
+or some of our available API endpoints but webhook requires less effort than polling an API since it allows near real time updates.
+
+# Available webhooks
+
+As of now, the webhooks available with us fall under the following categories.
+
+## JOB
```

Comment:
since it is a single category, just combine it with above. Name it Webhook types or something